README.md

```diff
@@ -0,0 +1,44 @@
+## Percona XtraBackup
+
+![PXB logo](storage/innobase/xtrabackup/doc/source/percona-xtrabackup-logo.jpg)
+
+Percona XtraBackup is an open-source hot backup utility for MySQL - based servers that doesn’t lock your database during the backup.
+
+Whether it is a 24x7 highly loaded server or a low-transaction-volume environment, Percona XtraBackup is designed to make backups a seamless procedure without disrupting the performance of the server in a production environment. Commercial support contracts are available.
+
+Percona XtraBackup 8.0 can back up data from InnoDB, XtraDB, MyISAM, and MyRocks tables on MySQL 8.0 servers as well as Percona Server for MySQL with XtraDB, Percona Server for MySQL 8.0, and Percona XtraDB Cluster 8.0. 
+
+Percona XtraBackup 2.4 can back up data from InnoDB, XtraDB, and MyISAM tables on MySQL 5.1, 5.5, 5.6 and 5.7 servers, as well as Percona Server for MySQL with XtraDB.
```

Comment:
Not sure if we want or need to mention 5.1 and 5.5 as they are very EOL and support will be dropped on the first sign of trouble maintaining compatibility.  5.6 is almost in the same boat but we will still make a best effort to maintain support for a little while.

Reply:
Fixed.